string vaisala_action
---
string vaisala_response
bool vaisala_error